shared/panels/_flows/request-payment-flow/_panels/rewards-selection/rewards-selection.tsx

```diff
@@ -0,0 +1,146 @@
+import { BillingProfileReactQueryAdapter } from "@/core/application/react-query-adapter/billing-profile";
+
+import { Button } from "@/design-system/atoms/button/variants/button-default";
+import { Skeleton } from "@/design-system/atoms/skeleton";
+
+import { EmptyStateLite } from "@/shared/components/empty-state-lite/empty-state-lite";
+import { SidePanelBody } from "@/shared/features/side-panels/side-panel-body/side-panel-body";
+import { SidePanelFooter } from "@/shared/features/side-panels/side-panel-footer/side-panel-footer";
+import { SidePanelHeader } from "@/shared/features/side-panels/side-panel-header/side-panel-header";
+import { useSidePanel } from "@/shared/features/side-panels/side-panel/side-panel";
+import { BillingProfileAccordion } from "@/shared/panels/_flows/request-payment-flow/_panels/_components/billing-profile-accordion/billing-profile-accordion";
+import { RewardsCardsSelection } from "@/shared/panels/_flows/request-payment-flow/_panels/_components/rewards-cards-selection/rewards-cards-selection";
+import { useRewardsSelectionPanel } from "@/shared/panels/_flows/request-payment-flow/_panels/rewards-selection/rewards-selection.hooks";
+import { useRequestPaymentFlow } from "@/shared/panels/_flows/request-payment-flow/request-payment-flow.context";
+
+export function Content() {
+  const { billingProfileId, rewardIds } = useRequestPaymentFlow();
```

Comment:
```suggestion
  const { billingProfileId = "", rewardIds } = useRequestPaymentFlow();
```

Could then remove fallbacks